service/gcsutils/remotefs/remotefs_os.go

```diff
@@ -0,0 +1,74 @@
+package remotefs
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+	"strconv"
+)
+
+// TODO: @gupta-ak. Continue adding as needed to support more docker functionality.
+
```

Comment:
Are there any more commands to support other then the following already implemented ones?
"stat" "lstat" "mkdirall" "resolvepath" "extractarchive" "archivepath
If there are, we should list them here, otherwise we can remove this comment.